### PR TITLE
Update readme with models section

### PR DESCRIPTION
**Pull request by AyushExel**
@Laughing-q can you take a look and tell me if I should add anything?

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Ultralytics updates documentations for new and existing pre-trained YOLO model architectures.

### 📊 Key Changes
- Added a new "Model Architectures" section in the README file.
- Provided comprehensive lists and details for the full range of YOLOv8 and YOLOv5u model variants.
- Included performance benchmarks for various models concerning speed, accuracy, and computational requirements.
- Outlined steps for community contributions to add more model architectures.

### 🎯 Purpose & Impact
- **Enhanced Clarity:** Users now have a centralized section to explore different YOLO model architectures.
- **Improved Accessibility:** Easy access to model performance metrics aids in selecting the right model for specific needs.
- **Community Growth:** Encouraging community contributions supports broader engagement and enhances the model ecosystem.
- Potential Impact: Users can make more informed decisions on model selection, potentially leading to better performance and efficiency in their applications. 🚀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,3 +276,86 @@ For professional support please [Contact Us](https://ultralytics.com/contact).
...
+
+# <div align="center">Appendix</div>
+
+## Model Architectures
+
+Ultralytics supports many model architectures. Visit [models](#) page to view detailed information and usage.
+Any of these models can be used by loading their configs or pretrained checkpoints if available.
+
+```bash
+> yolo model=model.yaml ..
```
Comment (Laughing-q) [inline README.md:288]: probably remove the `>` here?

File: README.md [hunk 2]
```diff
@@ -271,3 +276,86 @@ For professional support please [Contact Us](https://ultralytics.com/contact).
...
+### 2. YOLOv5u
+
+**About** - Anchor-free YOLOv5 models with new detection head and better speed-accuracy tradeoff </br>
+**Citation** -
+Available Models:
+
+- Detection - `yolov5nu`, `yolov5su`, `yolov5mu`, `yolov5lu`, `yolov5xu`
+
+<details><summary>Performance</summary>
+### Detection
```
Comment (Laughing-q) [inline README.md:353]: add an enter here or it looks like:
image

File: README.md (hunk 1 above)
Comment (AyushExel) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (AyushExel) [reply README.md:354]: done

**Comment by Laughing-q** [comment]
@AyushExel looks good to me! :)

**Comment by glenn-jocher** [comment]
@AyushExel can you move these changes to models/README?

**Comment by AyushExel** [comment]
@glenn-jocher done. We need a dedicated models page on docs with more details and comparison but let's take small incremental steps

**Comment by glenn-jocher** [comment]
@AyushExel agreed. PR merged!

**Pull request merged**